tasks/main.yml

```diff
@@ -17,7 +17,7 @@
   stat:
     path: "{{ linuxbrew_prefix_user }}/bin/brew"
   register: linuxbrew_user_st
-  become: false
+  become: true
```

Comment:
I'm not sure if this change should be included, but it was required for me

Reply:
I think this doesn't make sense.  `linuxbrew_prefix_user` is gathered based on the SSH user (e.g., "ubuntu").  So it makes sense to check for the existence of that path with the same user (e.g., "ubuntu").

Reply:
This one is checking if the installation exists in the current user home.